package.json

```diff
@@ -1,7 +1,11 @@
 {
   "name": "mine-support",
   "version": "0.0.1",
-  "description": "",
+  "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
+  "repository": {
+    "type" : "git",
+    "url" : "https://github.com/DEFRA/mine-support.git"
+  },
```

Comment:
While you're making updates to this file, might as well put the correct licence value in as well. I've been setting the author value differently, and adding a contributors section, and "homepage" is also a recommended item.

  "homepage": "https://github.com/DEFRA/mine-support",
  "author": "Defra DDTS",
  "contributors": [
    "John Watson <john.watson2@rpa.gov.uk>",
    "Mark Harrop <mark.harrop@defra.gov.uk>",
    "Matthew Collins <matthew.collins@defra.gov.uk>",
    "David Bingham <david.bingham@defra.gov.uk>"
  ],
  "license": "OGL-UK-3.0",